packages/atlas/src/components/_crt/CrtStatusWidget/CrtStatusWidget.tsx

```diff
@@ -0,0 +1,121 @@
+import { FC, useState } from 'react'
+
+import { Information } from '@/components/Information'
+import { JoyTokenIcon } from '@/components/JoyTokenIcon'
+import { NumberFormat, NumberFormatProps } from '@/components/NumberFormat'
+import { Text } from '@/components/Text'
+import { ExpandButton } from '@/components/_buttons/ExpandButton'
+import { DetailsContent } from '@/components/_nft/NftTile'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { formatDate } from '@/utils/time'
+
+import {
+  DetailsBox,
+  ExpandableContainer,
+  LabelText,
+  StatisticsContainer,
+  SupplyLine,
+  Widget,
+} from './CrtStatusWidget.styles'
+
+type Amount = NumberFormatProps['value']
+export type CrtStatusWidgetProps = {
+  name: string
+  creationDate: Date
+  supply: Amount
+  marketCap: Amount
+  revenue: Amount
+  revenueShare: Amount
+  transactionVolume: Amount
+}
+
+export const CrtStatusWidget: FC<CrtStatusWidgetProps> = ({
+  name,
+  creationDate,
+  supply,
+  marketCap,
+  revenue,
+  revenueShare,
+  transactionVolume,
+}) => {
+  const [isExpanded, expand] = useState(true)
+  const smMatch = useMediaMatch('sm')
+
+  const ticker = `$${name}`
+
+  return (
+    <Widget
+      title="Status"
+      customNode={
+        <>
+          <Text as="h4" variant="h500">
+            No active sale
+          </Text>
+
+          <SupplyLine>
+            <LabelText as="span" variant="t100">
+              Total supply:
+            </LabelText>
+            <NumberFormat as="span" variant="t100" format="short" value={supply} customTicker={ticker} withToken />
+            <Information />
+          </SupplyLine>
+
+          <StatisticsContainer onClick={() => expand(!isExpanded)}>
```

Comment:
It would be good to animate it.

There is pretty nice approach in `packages/atlas/src/views/studio/MyUploadsView/UploadStatusGroup/UploadStatusGroup.tsx` line 182 or in `NftWidget`.

Reply:
Yes that's a really nice one!